js/blogtini.js

```diff
@@ -314,21 +341,24 @@ async function main() {
   if (!STORAGE.base)
     STORAGE.base = base
 
-  tmp = yml.load(await fetcher(`${state.top_dir}config.yml`))
-  if (tmp)
+  const configYamlUrl = `${SITE_ROOT_BASE_URL}config.yml`
+  tmp = yml.load(await fetcher(configYamlUrl))
+  if (tmp) {
     cfg = { ...cfg, ...tmp } // xxx deep merge `sidebar` value hashmap, too
+  }
 
 
   log({
     filter_post, base: STORAGE.base, STORAGE_KEY, cfg, state,
   })
 
-  const prefix = cfg.repo === 'blogtini' ? state.pathrel : 'https://blogtini.com/'
-  // eslint-disable-next-line no-use-before-define
-  add_css(`${prefix}css/blogtini.css`) // xxxx theme.css
+  add_css(`${SITE_ROOT_BASE_URL}css/blogtini.css`) // xxxx theme.css
 
+  cleanUpInitialPayloadMarkup(document)
```

Comment:
This is to avoid what's happening a few lines below. To trash everything. See [js/utils **cleanUpInitialPayloadMarkup**](https://github.com/renoirb/blogtini/blob/8a8becc9/js/utils.mjs#L87-L98)